docker-compose.yml

```diff
@@ -0,0 +1,79 @@
+version: "3.8"
+services:
+  db:
+    image: postgres:14-alpine
+    networks:
+      new:
+        aliases:
+          - database
+    env_file:
```

Comment:
is there a postgres option to have all persisted data encrypted?

Reply:
Hmm definitely something to look into in the future but I don't know